source/reference/known-issues.txt

```diff
@@ -0,0 +1,56 @@
+=====================
+Known Issues for |bi|
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+SQL Compatibility Issues
+------------------------
+
+|bi-short| Returns Empty Set Where MySQL Returns ``NULL`` Values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some `SQL aggregate functions <https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html>`__
+(like ``SUM``, ``MIN``, or ``COUNT``) return ``NULL`` if there are no
+matching values for that aggregate function. This response is
+effectively an empty row.
+
+Given the exact same SQL query, the |bi-short| does not return
+``NULL`` for any aggregate functions; it returns an empty set instead.
+
+.. example::
+
+   Using Tableau, you query a MySQL database of students to find the
```

Comment:
This example uses MySQL and MongoDB databases. We may want to remove this mention of MySQL and move it closer to the MySQL `NULL` result code block.

Reply:
N/A. Fixed per suggestion below.